var/spack/repos/builtin/packages/cbtf-krell/package.py

```diff
@@ -263,3 +267,49 @@ def cmake_args(self):
             self.set_cray_login_node_cmake_options(spec, cmake_args)
 
         return cmake_args
+
+    def setup_environment(self, spack_env, run_env):
+        """Set up the compile and runtime environments for a package."""
+
+        # Environment settings for cbtf-krell
+        run_env.prepend_path('PATH', self.prefix.bin)
```

Comment:
This line should already be automatically added to the module file, so you can get rid of it.

Reply:
Ok.   Took this out.